### PR TITLE
Feature: Add possibility to add svg path to svgoConfig in svgLoader 

### PR DESCRIPTION
**Pull request by AndersLauridsen**
Hi, 
Great plugin you have made. 

I have made a small extra plugin which is used in combination with yours. Unfortunately, I can't find a way to access the svg filename path within that plugin, since the path is not added to svgoConfig. 

I have proposed to include an additional boolean as parameter to the svgLoader function to overwrite or set the path in svgoConfig. 

//Anders

### CODE REVIEW

**Comment by jpkleemans** [comment]
Hi, thanks for your PR. Why not always set the path? What would be the downsides?

**Comment by AndersLauridsen** [comment]
Hi, 
I can't think of any use case on top of my head, but currently is would be possible to set an alternative path when implementing the plugin, if that would be of interest for anyone.

**Comment by AndersLauridsen** [comment]
So in other words, I can't see any obvious reasons not to always set the path.

**Comment by jpkleemans** [comment]
Hi, I agree. It would be nice if you could set the path through the config, but if you didn't then the path would be set automatically.

Can you update the PR?

**Comment by AndersLauridsen** [comment]
Now, in the last PR the path can still be set in the config.  
If the path is not set, the svgoConfig.path is updated with the current svg file path and deleted before processing the next svg file.

File: index.js [hunk 1]
```diff
@@ -29,7 +29,13 @@ module.exports = function svgLoader (options = {}) {
       if (query === 'raw') {
         return `export default ${JSON.stringify(svg)}`
       }
-
+      
+      let deletePath = false;
+      if (svgoConfig !== false && svgoConfig.path === undefined){
```
Comment (jpkleemans) [inline index.js:34]:
```suggestion
      if (!svgoConfig && !svgoConfig.path){
```

File: index.js [hunk 2]
```diff
@@ -40,6 +46,10 @@ module.exports = function svgLoader (options = {}) {
         filename: path,
         transformAssetUrls: false
       })
+      
+      if (deletePath) {
+        delete svgoConfig.path;
```
[the remark this answers is not in the document]
Comment (AndersLauridsen) [reply index.js:51]: When testing in my own application, the svgoConfig.path set in for svg file n was still set when processing svg file n+1. 

So to be able to make the check "!svgoConfig.path" to set the path for the current svg file, while also deferring from overwriting a manually set path, svgoConfig.path needs to be set to 'undefined' again before processing the next svg file.

**Comment by AndersLauridsen** [comment]
Thanks for the small revision, and I hope the comment makes sense.

**Comment by jpkleemans** [comment]
Thanks for your clarification!
Would it be possible for you to add a (or more) Cypress tests for this new option?

**Comment by AndersLauridsen** [comment]
I have not been working with cypress tests before, and after digging into it, I can't seem to get my head around how get access to the content of the svgoConfig.path value during execution of the load function. 

The tests cases themselfes are pretty clear, though: 
  1) check if path if set in load() if svgoConfig.path is undefined. 
   2) check that path is not overwritten, if svgoConfig.path is set initially by the user. 

If you can see a way to acess this info during test, I will happily try to implement it.

**Comment by jpkleemans** [comment]
This is fixed in [v3.6.0](https://github.com/jpkleemans/vite-svg-loader/releases/tag/3.6.0)

**Pull request closed without merging**